src/components/Tako.tsx

```diff
@@ -71,3 +71,23 @@ export const Tako = () => {
     </div>
   )
 }
+
+export const TakoLogo = () => (
+  <div
+    style={{
+      margin: 15,
+      width: 40,
+      height: 40,
+      backgroundImage: `url(https://github.com/brumm/tako/blob/master/src/assets/icon-48.png?raw=true)`,
```

Comment:
this is the weakest part of this PR. I had some trouble adding the svg assets to the extension and ended up calling it a day with this url, but it should probably be changed to something more appropiate.

Reply:
Thanks so much for getting this started! I managed to get svg imports to work here: [`8fc5a96` (#78)](https://github.com/brumm/tako/pull/78/commits/8fc5a9604c4afe54a4e34e3a65a058778d8af95c)